extensions/amp-pan-zoom/0.1/test/test-amp-pan-zoom.js

```diff
@@ -250,28 +251,109 @@ describes.realWin('amp-pan-zoom', {
   });
 
   describe('gestures', () => {
-    it('should pan correctly via mouse when zoomed', () => {
+    it('should pan correctly via mouse when zoomed', async function() {
+      await getPanZoom();
+      await el.layoutCallback();
+      await impl.transform(0, 0, 2);
+      expect(svg.style.transform).to.equal('translate(0px, 0px) scale(2)');
+      const mouseDown = createPointerEvent('mousedown', 10, 10);
+      const mouseMove = createPointerEvent('mousemove', 20, 20);
+      el.dispatchEvent(mouseDown);
+      el.dispatchEvent(mouseMove);
+      await timeout(500);
+      expect(svg.style.transform).to.equal('translate(10px, 10px) scale(2)');
+    });
+  });
+
+  describe('transformEnd', () => {
+    it('should trigger only once after double tap zoom', () => {
+      let transformEndSpy;
       return getPanZoom()
           .then(() => el.layoutCallback())
-          .then(() => impl.transform(0, 0, 2))
           .then(() => {
-            expect(svg.style.transform).to
-                .equal('translate(0px, 0px) scale(2)');
-
-            const mouseDown = createPointerEvent('mousedown', 10, 10);
-            const mouseMove = createPointerEvent('mousemove', 20, 20);
-            const mouseUp = createPointerEvent('mouseup', 20, 20);
-            const transformEndPromise = listenOncePromise(el, 'transformEnd');
```

Comment:
Encountered a super weird issue here where the mouseup callback would never trigger and dev console listener subscriptions seemed to indicate that the mouseup event was never triggered. It seems that the transformEnd was previously triggered on mousemove.

Reply:
Many thanks to @sparhami who figured out that this was because the event wasn't bubbling and the listener was on the window.